invenio_drafts_resources/services/records/components/files.py

```diff
@@ -9,50 +9,50 @@
 # details.
 
 """Records service component base classes."""
-
 from invenio_i18n import gettext as _
-from invenio_records_resources.services.files.transfer import TransferType
-from invenio_records_resources.services.records.components import FilesOptionsComponent
-from marshmallow import ValidationError
 
-from .base import ServiceComponent
+from .files_base import RecordFilesComponent
+from invenio_records_resources.services.records.components import (
+    MediaFilesOptionsComponent,
+)
+from marshmallow import ValidationError
 
 
-class DraftFilesComponent(ServiceComponent):
+class DraftFilesComponent(RecordFilesComponent):
```

Comment:
all the methods were moved to files_base class (as default implementation/bahaviour for record-file component)